### ISSUE TITLE
Hide daily raids that have been completed

### ISSUE THREAD

**Issue by perhaps-pie** [recnaleerf/Ancheera#8]
Similar to how daily misc / co-op / casino hides rows when completed it'd be nice to have the same thing for raids but where it hides say "Tiamat Omega" when your alloted 3 for the day are done.

Maybe a checkbox option.

**Comment by recnaleerf** [comment]
Do you find value in grouping these also? I was thinking about grouping them by difficulty.

* Normal
* Hard
* EX/Omega
* Tier 2 (or whatever we call DAOs, Athenas, etc.)
* HL

That way, they can more easily "hide" like how "Monthly" section works.

**Comment by perhaps-pie** [comment]
Yes, actually I think that'd be great. It may even enable you to remove the per raid text label.

The current list is a bit unmanageable if you select 20+ raids.

**Comment by recnaleerf** [comment]
![raids](https://user-images.githubusercontent.com/29151267/27809737-2e4afb86-6007-11e7-9503-2b04f415b552.png)
